addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/PiiScanRuleUnitTest.java

```diff
@@ -191,6 +240,29 @@ void shouldNotRaiseAlertOnCssResponse() throws Exception {
         assertThat(alertsRaised.size(), is(0));
     }
 
+    @Test
+    void shouldNotRaiseAlertOnImageRequest() throws Exception {
+        // Given
+        HttpMessage msg = createMsg("");
```

Comment:
I guess this one and the one below should actually have content, but I had raised this as separate commits on purpose. If someone can give me a tip on going back and fixing the first of the two then I'm happy to tweak it